src/components/Composer/index.js

```diff
@@ -11,6 +11,8 @@ import CONST from '../../CONST';
 import updateIsFullComposerAvailable from '../../libs/ComposerUtils/updateIsFullComposerAvailable';
 import getNumberOfLines from '../../libs/ComposerUtils/index';
 
+const COPY_DROP_EFFECT = 'copy';
```

Comment:
`propTypes` should be the first constant in the file. This should either be defined below the `propTypes` and `defaultProps` or in `CONST.js`.